Tests.Template/integration_template.ps1

```diff
@@ -17,17 +17,18 @@ $Global:DSCModuleName      = 'x<ModuleName>' # Example xNetworking
 $Global:DSCResourceName    = 'MSFT_x<ResourceName>' # Example MSFT_xFirewall
 # /TODO
 
-#region HEADER
-if ( (-not (Test-Path -Path '.\DSCResource.Tests\')) -or `
-     (-not (Test-Path -Path '.\DSCResource.Tests\TestHelper.psm1')) )
+#region HEADER (V2)
```

Comment:
I added the V2 mainly so that it would be more obvious if a test script contained an older version header and also so that it would be easier to write code to automatically update this test header/footer. - But this isn't strictly necessary. Happy to remove it if you think!